scalyr_agent/util.py

```diff
@@ -99,6 +104,39 @@
 # serialized values easier.
 SORT_KEYS = False
 
+# Maps user-friendly string we expose in the configuration to the internal Python module name
+SUPPORTED_COMPRESSION_ALGORITHMS = [
+    "deflate",
+    "bz2",
+]
+
+# lz4 and zstandard library is not available for Python 2.5
```

Comment:
I think you mean Python 2.6

Reply:
Yep, fixed - d7f2fada34691dc5010e7839994c2e6474ab1fc6.